src/frontend/api/yahooFinance.tsx

```diff
@@ -1,7 +1,7 @@
-export const fetchNews = async (stock: string): Promise<object> => {
+export const fetchNews = async (stock: string): Promise<string> => {
   const api: string = process.env.REACT_APP_YAHOO_FINANCE_API_KEY || ''
-  
-  let res: object = await fetch(
+
+  const result: string = await fetch(
```

Comment:
Since we're using async/await, I recommend not using the .then() syntax. Something like this would be cleaner:
```javascript
const result: string = await fetch(
    'https://apidojo-yahoo-finance-v1.p.rapidapi.com/auto-complete?region=US&q=' + stock,
    {
      method: 'GET',
      headers: {
        'x-rapidapi-host': 'apidojo-yahoo-finance-v1.p.rapidapi.com',
        'x-rapidapi-key': api,
      }
    }
  ).catch(err => console.log(err)

if (result.status !== 200) {
  throw new Error('Something wrong with API Dojo Yahoo Finance v1 API')
}
return result.json()
```